eng/Versions.props

```diff
@@ -8,6 +8,7 @@
     <UsingToolXliff>false</UsingToolXliff>
   </PropertyGroup>
   <PropertyGroup>
+    <HumanizerCoreVersion>2.14.1</HumanizerCoreVersion>
```

Comment:
Do we want to automatically lift this version if SBE is updated or should we stay on the version declared by M.CA?  There is precedence for staying on the declared versions.  Sys.Comp is the only exception.

Reply:
Wouldn't it cause a prebuilt to reference the originally declared version?